jboss/container/wildfly/launch/datasources/module.yaml

```diff
@@ -9,4 +9,60 @@ modules:
 
 execute:
 - script: configure.sh
-  user: '185'
\ No newline at end of file
+  user: '185'
+
+envs:
+    - name: "DB_SERVICE_PREFIX_MAPPING"
+      description: "It is used to define the mapping prefixes for internal datasources. The allowed value for this variable is a comma-separated list of POOLNAME-DATABASETYPE=PREFIX triplets, where POOLNAME is used as the pool-name in the datasource, DATABASETYPE is the database driver to use, PREFIX is the prefix used in the names of environment variables that are used to configure the datasource."
+      example: "true"
```

Comment:
Seems a wrong example.

Reply:
I can see some others.

Reply:
Right, that's incorrect

Reply:
@jfdenise I've just added an example with a description.